evennia/objects/objects.py

```diff
@@ -73,10 +73,16 @@ def _recache(self):
         self._sessid_cache = list(
             set(int(val) for val in (self.obj.db_sessid or "").split(",") if val)
         )
-        if any(sessid for sessid in self._sessid_cache if sessid not in evennia.SESSION_HANDLER):
+        if any(
+            sessid
+            for sessid in self._sessid_cache
+            if sessid not in evennia.SESSION_HANDLER
+        ):
```

Comment:
Please set up your IDE to follow Evennia line width standard (100 characters). That makes the PR smaller (easier to review) and clearer since you only see the important changes.